tests/template/test_com_template.py

```diff
@@ -1,14 +1,13 @@
 import io
 import os
-import pytest
+
 import numpy as np
 import nbformat
 from temp_utils import _get_hdf5_params
 from libertem.web.notebook_generator.notebook_generator import notebook_generator
 from nbconvert.preprocessors import ExecutePreprocessor
 
 
-@pytest.mark.skip(reason="requires `empyre`")
```

Comment:
The same needs to be done in `test_radial_fourier_template`

Reply:
Thx! I'll look into that later. Just saw the note from @twentyse7en above, will follow up on that.